bindings/pydeck/docs/CHANGELOG.rst

```diff
@@ -7,7 +7,7 @@ Releases and associated GitHub PRs for pydeck are documented here.
 ------------
 
 0.9.0b0 - Apr 24 2024
-^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Should we mention the limitations in the changelog? What file are users most likely to consult?

Reply:
Yes, good point, I've added a changelog entry for pydeck v0.9, and a note about the lack of Jupyter-specific features. I've tried to add a warning on every relevant page, but I suspect that [jupyter-specific features](https://deckgl.readthedocs.io/en/latest/jupyter.html) and the installation page are the main ones.